app/models/user.rb

```diff
@@ -0,0 +1,9 @@
+class User < ApplicationRecord
+  has_many :microposts, dependent: :destroy
+
+  validates :name, presence: true,
+length: {maximum: Settings.models.user.name_max_length}
+
+  validates :email, presence: true,
+length: {maximum: Settings.models.user.email_max_length}
```

Comment:
```suggestion
  validates :name, presence: true,
length: {maximum: Settings.models.user.name_max_length}
  validates :email, presence: true,
length: {maximum: Settings.models.user.email_max_length}
```
Các thành phần cùng chức năng thì k cần line trống đâu